File: messageGo/grpc/methodData/client_proto_msg.go
package methodData

// agent service forward client message request
type ClientProtoMessageInput struct {
	MsgVersion    int64  `json:"msgVersion"`    // 消息版本号 值为毫秒时间戳
	AgentAppId    string `json:"agentAppId"`    // 网关 appId
	UserId        int64  `json:"userId"`        //
	ProtoBytesReq []byte `json:"protoBytesReq"` // proto message marshal bytes
}

// agent service forward client message response
type ClientProtoMessageOutput struct {
	MsgVersion     int64  `json:"msgVersion"`     // 消息版本号 值为毫秒时间戳
	AgentAppId     string `json:"agentAppId"`     // 网关 appId
	UserId         int64  `json:"userId"`         //
	ProtoBytesResp []byte `json:"protoBytesResp"` // proto message marshal bytes
}

File: messageDotNet/grpc/MethodData/ClientProtoMsg.cs
using System;
using UnityEngine;

/// <summary>
/// agent service forward client message request
/// </summary>
[Serializable]
internal class ClientProtoMessageInput
{
    // 消息版本号 值为毫秒时间戳
    public long MsgVersion;
    public string AgentAppId;
    public long UserId;
    public byte[] ProtoBytesReq;

    public string ToJson()
    {
        return JsonUtility.ToJson(this);
    }
}


/// <summary>
/// agent service forward client message response
/// </summary>
[Serializable]
internal class ClientProtoMessageOutput
{
    // 消息版本号 值为毫秒时间戳
    public long MsgVersion;
    public string AgentAppId;
    public long UserId;
    public byte[] ProtoBytesResp;

    public string ToJson()
    {
        return JsonUtility.ToJson(this);
    }

}